jboss-tck-runner/pom.xml

```diff
@@ -490,8 +490,8 @@
                                     <outputDirectory>${JBOSS_HOME}/standalone/configuration</outputDirectory>
                                     <resources>
                                         <resource>
-                                            <directory>${JBOSS_HOME}/docs/examples/configs</directory>
-                                            <includes>standalone-activemq-embedded.xml</includes>
+                                            <directory>${JBOSS_HOME}/standalone/configuration</directory>
```

Comment:
We should just delete this whole plugin execution. There is no need to copy the same file to itself.

Reply:
That makes sense, I thought there was some magic somewhere that was causing this to copy to `standalone.xml` but removing the copy makes more sense.

Reply:
Done